### PR TITLE
feat(cli): updates

### PR DESCRIPTION
**Pull request by noahlitvin**

### CODE REVIEW

File: packages/cli/src/settings.ts [hunk 1]
```diff
@@ -36,6 +36,11 @@ export type CliSettings = {
    */
   ipfsTimeout?: number;
 
+  /**
+   * DEPRECATED: fallback for readIpfsUrl
```
Comment (mjlescano) [inline packages/cli/src/settings.ts:40]:
```suggestion
   * @deprecated: fallback for readIpfsUrl
```

**Review by dbeal-eth — APPROVED** [review]
Technically this doesn't handle the case where user has a config file with `ipfsUrl` defined it looks like, only the envvar

**Comment by noahlitvin** [comment]
Can we just leave readIpfsUrl as ipfsUrl?

File: packages/cli/src/loader.test.ts [hunk 2]
```diff
@@ -53,7 +53,7 @@ describe('getMainLoader', function getMainLoaderTestCases() {
     const settings: CliSettings = {
       providerUrl: '',
       registries: [],
-      readIpfsUrl: 'ipfs',
+      ipfsUrl: 'ipfs',
```
Comment (noahlitvin) [inline packages/cli/src/loader.test.ts:56]: This is what's on main? Seems weird it's just `ipfs` here?

**Pull request merged**